### PR TITLE
Refoldering

### PR DESCRIPTION
**Pull request by CommanderStorm**
## Proposed Changes (include Screenshots if possible)

- This PR refactors every microservice to live under the `server` folder.

## How to test this PR

1. See if this is a good change
2. Testing in production if the simplified deployment system holds its ground.

## How has this been tested?

- internal monolouge

## Checklist:

- [x] I have updated the documentation / No need to update the documentation
- [x] I have run the linter

### CODE REVIEW

File: server/main-api/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+```bash
+mkdir -p data
+wget -P data https://nav.tum.de/cdn/api_data.json
+```
+
+Alternatively, you can run the `data` part of this project and generate this file by that part of our docs.
+To link the output directory to the server data directory, so that you don't need to copy on every update you can use:
+
+```bash
+ln -s ../data/output data
```
Comment (octycs) [inline server/main-api/README.md:44]:
```suggestion
ln -s ../../data/output data
```

File: server/main-api/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+# Set filterable attributes
+curl -X PUT 'http://localhost:7700/indexes/entries/settings/filterable-attributes' --data '["facet"]'
+
+# Upload entries data
+curl -i -X PUT 'http://localhost:7700/indexes/entries/documents' --header 'content-type: application/json' --data-binary @data/search_data.json
+
+# Configure index
+curl -X PUT 'http://localhost:7700/indexes/entries/settings/ranking-rules' --data '["words","typo","rank:desc","exactness","proximity","attribute"]'
+
+curl -X PUT 'http://localhost:7700/indexes/entries/settings/synonyms' --data @../data/search_synonyms.json
```
Comment (octycs) [inline server/main-api/README.md:92]:
```suggestion
curl -X PUT 'http://localhost:7700/indexes/entries/settings/synonyms' --data @../../data/search_synonyms.json
```

File: server/main-api/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+Of course documentation is one part of the process.
+Run API-Fuzz-Test and [schemathesis](https://github.com/schemathesis/schemathesis) on API Server to ensure specification is up-to-date and without holes.
+To do so, run the following commands against the API Server:
+
+```bash
+python -m venv venv
+source venv/bin/activate
+pip install schemathesis
+st run --workers=auto --base-url=http://localhost:8080 --checks=all ../openapi.yaml
```
Comment (octycs) [inline server/main-api/README.md:132]:
```suggestion
st run --workers=auto --base-url=http://localhost:8080 --checks=all ../../openapi.yaml
```

File: server/README.md [hunk 4]
```diff
@@ -1,156 +1,31 @@
...
-```bash
-curl -X DELETE 'http://localhost:7700/indexes/entries'
-```
-
-MeiliSearch provides an interactive interface at [http://localhost:7700](http://localhost:7700).
-
-### API-Changes
+ln -sf ../data/output/ data
+cd calendar && ln -sf ../../data/output/ data
+cd main-api && ln -sf ../data/output/ data
```
Comment (octycs) [inline server/README.md:28]:
```suggestion
cd calendar && ln -sf ../../data/output/ data && cd ..
cd main-api && ln -sf ../../data/output/ data
```
Shouldn't it be like this? Or do we need `data` mapped into the root `server` folder as well

**Review by octycs — APPROVED** [review]
It's only a few path updates in the documentation that I found. There's also one in `server/calendar/README.md` for the `data` link that I cound't do as review comment.

File: server/README.md (hunk 4 above)
Comment (CommanderStorm) [reply server/README.md:28]: I am going to push this to another PR, because this way it stays reviewable

**Pull request merged**